spec/requests/games_spec.rb

```diff
@@ -2,24 +2,34 @@
 
 RSpec.describe "Games", type: :request do
 
-  let(:user) { create :user }
+  let(:user)  { create :user }
+  let(:lobby) { create :lobby }
 
-  describe "GET /v1/games/:id" do
+  describe "GET /v1/lobbies/:id/games/:id" do
 
-    context "with a valid game id" do
-      let(:game) { create :game }
-      let(:path) { "/v1/games/#{game.id}" }
+    context "with a valid lobby and game id" do
+      let(:game)  { create :game, lobby: lobby }
+      let(:path)  { "/v1/lobbies/#{lobby.id}/games/#{game.id}" }
       before { get_with_access_token path, user.access_token }
 
       it "responds with game data" do
         expect(response).to have_http_status 200
         expect(json['data']['type']).to eq "games"
-        expect(json['data']['attributes']).to have_key("created_at")
+      end
+      it "responds with card data" do
+        expect(json['data']['attributes']).to have_key("board")
+        expect(json['data']['attributes']).to have_key("deck")
+        expect(json['data']['attributes']['board'].all? {
+          |card| card['attributes'].keys == [ "symbol", "number", "color", "shading" ]
```

Comment:
seems like overkill to test all keys for all cards in the requests, without testing any values. Perhaps you could test one card's keys and values. or perhaps you could lean on your serializer unit specs for most of the actual response specifics.